src/meta/app/src/share/share_meta_v2.rs

```diff
@@ -269,6 +275,25 @@ impl ShareMetaV2 {
                 }
                 _ => {}
             },
+
+            ShareObject::View(name, db_id, table_id) => {
+                if privileges == ShareGrantObjectPrivilege::Select {
+                    for table in &mut self.table {
+                        if table.table_id == *table_id {
```

Comment:
If my mind is correct, ShareObject store the db_id and table_id and object's origin name. 

If object rename to other name, the id is eq. So in display it will display object's origin name, but privilege check is use id. So will not return err?

Reply:
rename a share object will cause access to the share object lose effect